src/vivarium_public_health/disease/models.py

```diff
@@ -125,6 +124,49 @@ def setup(self, builder):
                                                         get_data_functions=get_data_functions)])
 
 
+class SIR_fixed_duration:
+
+    def __init__(self, cause: str, duration):
+        """
+        Parameters
+        ----------
+        cause
+        duration
+        """
+        self.cause = cause
+        if not isinstance(duration, pd.Timedelta):
+            self.duration = pd.Timedelta(days=float(duration) // 1, hours=(float(duration) % 1) * 24.0)
```

Comment:
I think we are going to regret this complexity!
```suggestion
            self.duration = pd.Timedelta(days=duration)
```